src/btree/bt_cursor.c

```diff
@@ -232,107 +232,195 @@ __cursor_fix_implicit(WT_BTREE *btree, WT_CURSOR_BTREE *cbt)
 }
 
 /*
- * __wt_cursor_valid --
- *     Return if the cursor references an valid key/value pair.
+ *  __cursor_valid_int_insert --
+ *     Check the insert list for a valid update.
  */
-int
-__wt_cursor_valid(
-  WT_CURSOR_BTREE *cbt, WT_ITEM *key, uint64_t recno, bool *valid, bool check_bounds)
+static int
+__cursor_valid_int_insert(WT_CURSOR_BTREE *cbt, WT_ITEM *key, bool *valid, bool check_bounds)
```

Comment:
Why it's called int_insert? I don't understand the meaning of int

Reply:
It means internal. I think without wt prefix, it already means an internal function, no need to repeat that.

Reply:
Its meant to indicate that it is an internal function and shouldn't be called by things outside of cursor valid.

Reply:
I could go either way so I'll remove the _int for now.